Tests/Orso.Arpa.Api.Tests/IntegrationTests/ProjectsControllerTests.cs

```diff
@@ -328,7 +328,7 @@ public async Task Should_Not_Create_Due_Wrong_Dates()
             validationProblemDetails.Errors.Should().BeEquivalentTo(new Dictionary<string, string[]>() { { "EndDate", new[] { "'EndDate' must be greater than 'StartDate'" } } });
         }
 
-        [Test, Order(1010)]
+        [Test, Order(105)]
```

Comment:
100 - 999   | test that modify value in the data base
1000 - 9999 | test that create new entries in the data base

add habe ich bisher zu create hinzugepackt (.. new entries in the data base..)